systemtests/utils/utils.go

```diff
@@ -154,6 +153,8 @@ func StartNetPlugin(t *testing.T, nodes []TestbedNode, nativeInteg bool) {
 
 // StartNetmasterWithFlags starts netplugin on specified testbed nodes with specified flags
 func StartNetmasterWithFlags(t *testing.T, node TestbedNode, flags map[string]string) {
+	time.Sleep(5 * time.Second)
```

Comment:
> I'm guessing that what you might be seeing is normal periodic leader election that would not occur before.

I remember seeing read request failures in consul logs with reason 'no leader' so I don't think it is due to a normal periodic election becasue I would assume these databases should stay available under normal conditions. I will retry and double check. We can go ahead with change for now.